### PR TITLE
Remove Blocks

### PR DESCRIPTION
**Pull request by sebmarkbage**
This concept was replaced by server components, so we don't need any of this anymore.

### CODE REVIEW

File: packages/react-reconciler/src/__tests__/ReactBlocks-test.js [hunk 1]
```diff
@@ -1,351 +0,0 @@
...
-
-    resolvePromises = () => {
-      const res = unresolved;
-      unresolved = [];
-      res.forEach(r => r());
-    };
-  });
-
-  // @gate experimental
-  it('renders a simple component', () => {
```
Comment (gaearon) [inline packages/react-reconciler/src/__tests__/ReactBlocks-test.js:61]: Nothing worth converting here?

**Review by gaearon — APPROVED** [review]
Screenshot 2020-10-31 at 00 25 41

File: packages/react-reconciler/src/__tests__/ReactBlocks-test.js (hunk 1 above)
Comment (sebmarkbage) [reply packages/react-reconciler/src/__tests__/ReactBlocks-test.js:61]: I don't think so. These are all just "client blocks" which is just components.

**Comment by sebmarkbage** [comment]
End of an era.

File: packages/react-devtools-shared/src/backend/renderer.js [hunk 2]
```diff
@@ -188,7 +187,7 @@ export function getInternalReactConstants(
       LazyComponent: 16,
       MemoComponent: 14,
       Mode: 8,
-      OffscreenComponent: 23, // Experimental
+      OffscreenComponent: 22, // Experimental
```
Comment (bvaughn) [inline packages/react-devtools-shared/src/backend/renderer.js:190]: This change looks wrong. `OffscreenComponent` is `22` in master now, but in 17.0.0 and 17.0.1 it's `23`.

Comment (bvaughn) [reply packages/react-devtools-shared/src/backend/renderer.js:190]: Fixed in #20362

**Pull request merged**